### ISSUE TITLE
TypeError: context.issue is not a function

### ISSUE THREAD

**Issue by ibrokemypie** [JasonEtco/todo#157]
Running todo locally to test for todo's results in an error
Command used is 

```
node ./bin/todo -o ibrokemypie -s 26a568141e96e1fee084dca17ba72df9e2ef43c8 -f https://github.com/ibrokemypie/makeppkg
```

Error is

```
TypeError: context.issue is not a function
    at module.exports (/Users/conor/git/todo/src/utils/chunk-diff.js:12:59)
    at module.exports (/Users/conor/git/todo/src/push-handler.js:24:5)
    at process._tickCallback (internal/process/next_tick.js:68:7)
    at Function.Module.runMain (internal/modules/cjs/loader.js:745:11)
    at startup (internal/bootstrap/node.js:279:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:696:3)
```

**Comment by ksomervi** [comment]
I get a similar error:

    $ node ./bin/todo.js -o ksomervi -r invader -f ../allegro/invader/src/fighter.cpp  
    TypeError: context.issue is not a function
        at module.exports (/home/ksomervi/src/todo/src/utils/chunk-diff.js:12:59)
        at module.exports (/home/ksomervi/src/todo/src/push-handler.js:24:5)
        at <anonymous>
        at process._tickCallback (internal/process/next_tick.js:189:7)
        at Function.Module.runMain (module.js:696:11)
        at startup (bootstrap_node.js:204:16)
        at bootstrap_node.js:625:3

I was hoping to look into issue #156

**Comment by JasonEtco** [comment]
Thanks folks - I believe that this should be closed by #165. If you're still having issues feel free to reopen ❤️
